uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/manager-notification-badge/components/manager-notification-badge.component.js

```diff
@@ -0,0 +1,68 @@
+/* eslint-env amd */
+
+define([
+  'common/lodash',
+  'leave-absences/manager-notification-badge/modules/components'
+], function (_, components) {
+  components.component('managerNotificationBadge', {
+    templateUrl: ['settings', function (settings) {
+      return settings.pathTpl + 'components/manager-notification-badge.html';
+    }],
+    controllerAs: 'managerNotificationBadge',
+    controller: ('ManagerNotificationBadgeController', ManagerNotificationBadgeController)
```

Comment:
I saw this syntax on the john papa's page but it seemed weird to me. The one suggested by you works, changed it.